elastic/kibana
# [EEM] Add built in definitions for hosts and containers

### Summary

This PR adds built in definitions for `hosts` and `containers` based on ECS data.

### How to test

1. Check out this branch of Kibana
2. Start ES and Kibana, verify that start up works and that the two definitions are installed (by calling `GET /internal/entities/definition` or checking that the transforms are installed).
3. Ingest some data for each definition to work with, verify that you get data in `entities-host-*` and `entities-container-*` and that the data matches the expected shape (metadata and metrics[1])

[[1]](https://github.com/elastic/kibana/pull/193157#issuecomment-2355609821)
